### ISSUE TITLE
[Query] Hiding rows with 'undefined' values

### ISSUE THREAD

**Issue by Natalique** [yesoreyeram/yesoreyeram-boomtable-panel#74]
is there a way i can define columns that if the value there us undefined the row would be hidden?

**Comment by balakrishnabk** [comment]
i also have a similar query. use multiple options for *hide* field.  My use case is to hide rows with 0 and "undefined" cell values.

**Comment by tobi4s** [comment]
Yes I can confirm. Same issue. I can't hide values, which are the number `0`. 

My workaround: Add 1 to each value (at DB query) and hide values below "threshold + 1".

**Comment by PavelvKalinin** [comment]
not everywhere you can do +0.1
Empty values are not overridden by "Null Value"

**Comment by igorantolic** [comment]
SOLUTION IS HIDDEN :)
Yes THERE ARE "NEWS" -> [PavelvKalinin] gave the solution but you did not spend enough time reading
In query just add a dummy value like in example bellow: +0.001
Whatever number that does not influence the numbers significantly
Then null become +0.001 so use Hide series /Hide if value is below: 1 for example 

sort_desc(
    sum by (uri,status,outcome)(
      (
        rate (
            http_server_requests_seconds_count {app=~"web-gateway", outcome="SUCCESS", uri!="/health" }
            [10m]
        )*60+0.001
      ) 
    )
)
